### PR TITLE
People Update KY

### PR DESCRIPTION
**Pull request by bfossen-ce**
People Update KY

### CODE REVIEW

File: data/ky/retired/Josie-Raymond-2973d2c4-1ca8-4842-a3d9-d5444b212c00.yml [hunk 1]
```diff
@@ -3,30 +3,32 @@ name: Josie Raymond
 given_name: Josie
 family_name: Raymond
 email: Josie.Raymond@lrc.ky.gov
-image: https://legislature.ky.gov/Legislators%20Thumbnail%20Images/house31.jpg
+image: https://legislature.ky.gov/Legislators%20Thumbnail%20Images/house41.jpg
 party:
 - name: Democratic
```
Comment (maxknutsen) [inline data/ky/retired/Josie-Raymond-2973d2c4-1ca8-4842-a3d9-d5444b212c00.yml:8]: @bfossen-ce this person needs to be moved from retired to legislature

**Review by maxknutsen — COMMENTED** [review]
@bfossen-ce only one change needed here. Josie Raymond should be moved from retired to legislature.

**Comment by bfossen-ce** [comment]
> @bfossen-ce only one change needed here. Josie Raymond should be moved from retired to legislature.

This has been done.

**Review by maxknutsen — APPROVED** [review]
Looks good!

**Pull request merged**